### PR TITLE
Removed all `model` input in `RobotModelObject.__init__`

### PR DESCRIPTION
**Pull request by yck011522**
I'm not sure since which version of compas that the Scene will no longer pass model as parameter to the __init__ of RobotModelObject. This is pass as `item` now. The model is set at `SceneObject.item` and previously @PingHsunTsai have already made it available at `BaseRobotModelObject.model`. 

That change had not been updated to the other RobotModelObjects at Rhino GH and blender. So here it is.

This fixes the error message:
```
Message: init() takes exactly 2 arguments (4 given)

Traceback:
line 116, in add, “C:\Users\alici\AppData\Roaming\McNeel\Rhinoceros\7.0\scripts\compas\scene\scene.py”
line 10, in , “C:\Users\alici\AppData\Local\Temp\TempScript.py”
```

### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/compas_robots/ghpython/scene/robotmodelobject.py
Comment (gonzalocasas) [inline src/compas_robots/ghpython/scene/robotmodelobject.py:26]: Actually, if we're not doing anything in these constructors, can't we just delete them entirely? All 3 of them?

Comment (yck011522) [reply src/compas_robots/ghpython/scene/robotmodelobject.py:26]: Up to you, but they do have a useful docstring. The RobotModelObject for Blender do have a useful __init__, so that's maybe one reason to be consistent? 

Your call.

Comment (gonzalocasas) [reply src/compas_robots/ghpython/scene/robotmodelobject.py:26]: Fair enough! "Clear docs" trumps "less code", let's keep them!

**Review by gonzalocasas — APPROVED** [review]
LGTM! If you tested that this works, then let's go ahead and merge/release!

**Pull request merged**
